src/main/scala/it/pagopa/interop/tenantprocess/error/ResponseHandlers.scala

```diff
@@ -192,4 +192,15 @@ object ResponseHandlers extends AkkaResponses {
       case Failure(ex: TenantByIdNotFound) => notFound(ex, logMessage)
       case Failure(ex)                     => internalServerError(ex, logMessage)
     }
+
+  def internalAssignCertifiedAttributeResponse[T](logMessage: String)(
+    success: T => Route
+  )(result: Try[T])(implicit contexts: Seq[(String, String)], logger: LoggerTakingImplicit[ContextFieldsToLog]): Route =
+    result match {
+      case Success(s)                                     => success(s)
+      case Failure(ex: CertifiedAttributeAlreadyInTenant) => badRequest(ex, logMessage)
```

Comment:
To comply with openapi spec
```suggestion
      case Failure(ex: CertifiedAttributeAlreadyInTenant) => conflict(ex, logMessage)
```